builtin/providers/azurerm/provider.go

```diff
@@ -85,7 +85,8 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_lb_probe":                resourceArmLoadBalancerProbe(),
 			"azurerm_lb_rule":                 resourceArmLoadBalancerRule(),
 
-			"azurerm_managed_disk": resourceArmManagedDisk(),
+			"azurerm_managed_disk":  resourceArmManagedDisk(),
+			"azurerm_managed_image": resourceArmManagedImage(),
```

Comment:
I think Jay has convinced me it's probably better to go with just azurerm_image since it really has nothing to do with managed disks.